src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/IncentivesService.kt

```diff
@@ -14,17 +14,17 @@ class IncentivesService(
   val incentivesWebClient: WebClient,
   @Value("\${api.incentives.timeout:20s}") val timeout: Duration
 ) {
-  fun getCurrentIncentive(bookingId: Long): CurrentIncentive? =
```

Comment:
renamed to avoid confusing name clash with the CurrentIncentive in the Prisoner domain class